packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -692,6 +693,12 @@ class SemanticsProperties extends DiagnosticableTree {
   /// text field.
   final bool textField;
 
+  /// If non-null, indicates that this subtree represents a slider.
+  ///
+  /// Talkback/\VoiceOver provides users with the hint "slider" when a
```

Comment:
Is this true? The engine PR doesn't seem to implement this flag for Android or iOS: https://github.com/flutter/engine/pull/20385/files